js/repl/Repl.js

```diff
@@ -317,7 +329,14 @@ class Repl extends React.Component {
         sourceMap: runtimePolyfillState.isEnabled,
         useBuiltIns: state.builtIns,
       })
-      .then(result => this.setState(result, setStateCallback));
+      .then(result => {
+        const { raw, compiled } = result.meta;
+        this._updateStatus({
+          rawSize: prettySize(raw.size),
+          compiledSize: prettySize(compiled.size),
+        });
```

Comment:
The `_updateStatus` method does nothing but call `setState` and it's only called here. Just combine this with the below `setState(result)` call?